js/arena.js

```diff
@@ -11,28 +11,28 @@ export default class Arena {
 this.gladiators.length <2 ? this.gladiators.push(newGlad) : console.log("You have Two gladiators");
     }
 
-
     fight(){
-       if(this.gladiators[0].weapon =="Trident" && this.gladiators[1].weapon =="Spear")
-       {
-           this.gladiators.pop();
-       }else if (this.gladiators[0].weapon =="Spear" && this.gladiators[1].weapon =="Club"){
-           this.gladiators.pop();
-       }
-
-       else if (this.gladiators[0].weapon =="Club" && this.gladiators[1].weapon =="Trident"){
-        this.gladiators.pop();
-    }
-    else if(this.gladiators[1].weapon =="Trident" && this.gladiators[0].weapon =="Spear")
-       {
-           this.gladiators.pop();
-       }else if (this.gladiators[1].weapon =="Spear" && this.gladiators[0].weapon =="Club"){
-           this.gladiators.pop();
-       }
-
-       else if (this.gladiators[1].weapon =="Club" && this.gladiators[0].weapon =="Trident"){
-        this.gladiators.pop();
-    }
-
-    }
-}
+        if(this.gladiators[0].weapon =="Trident" && this.gladiators[1].weapon =="Spear")
+        {
+            this.gladiators.shift();
```

Comment:
you need to remove the correct element in the array

pop() for the second element
and shift() to remove the first element